src/protocols/rdp/rdp.c

```diff
@@ -143,6 +142,7 @@ BOOL rdp_freerdp_pre_connect(freerdp* instance) {
     if (!gdi_init(instance, guac_rdp_get_native_pixel_format(FALSE)))
         return FALSE;
 
+#if 0
```

Comment:
Looks like I should wait for this stuff to be finalized before reviewing the protocols

Reply:
RDP and VNC should now be complete and no longer contain any of these `#if 0` blocks.

I'm now just finishing up the last terminal emulator bits and retesting.